test/fixtures/restaurants.yml

```diff
@@ -0,0 +1,21 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  new: MyString
+  edit: MyString
+  upgrade: MyString
+  delete: MyString
+  destroy: MyString
+  create: MyString
+  index: MyString
+  show: MyString
+
+two:
+  new: MyString
+  edit: MyString
+  upgrade: MyString
+  delete: MyString
+  destroy: MyString
+  create: MyString
+  index: MyString
+  show: MyString
```

Comment:
consider using [FactoryBot](https://github.com/thoughtbot/factory_bot) - a library for creating mocks that are derived from either fake or real data.
We use it in all of our ruby products and it's a good thing to get acquainted with.